ui/src/app/views/signed-in/component.ts

```diff
@@ -29,6 +31,7 @@ export class SignedInComponent implements OnInit {
   familyName = '';
   profileImage = '';
   sidenavToggle = false;
+  environment = environment;
```

Comment:
I think I'd slightly prefer just plumbing the publicUiUrl into the template rather than the whole environment object. I don't have any concrete objection aside from instinct and favoring minimum binding of dependencies.